gtsfm/two_view_estimator.py

```diff
@@ -257,56 +296,52 @@ def create_computation_graph(
                 keypoints_i1_graph,
                 keypoints_i2_graph,
                 pre_ba_v_corr_idxs,
+                putative_corr_idxs,
                 camera_intrinsics_i1_graph,
                 camera_intrinsics_i2_graph,
                 pre_ba_i2Ri1,
                 pre_ba_i2Ui1,
+                i2Ti1_prior,
             )
         else:
             post_ba_i2Ri1 = pre_ba_i2Ri1
             post_ba_i2Ui1 = pre_ba_i2Ui1
             post_ba_v_corr_idxs = pre_ba_v_corr_idxs
 
         # if we have the expected GT data, evaluate the computed relative pose
-        if gt_wTi1_graph is not None and gt_wTi2_graph is not None:
-            i2Ti1_expected_graph = gt_wTi2_graph.between(gt_wTi1_graph)
-            pre_ba_R_error_deg, pre_ba_U_error_deg = dask.delayed(compute_relative_pose_metrics, nout=2)(
-                pre_ba_i2Ri1, pre_ba_i2Ui1, i2Ti1_expected_graph
-            )
-            post_ba_R_error_deg, post_ba_U_error_deg = dask.delayed(compute_relative_pose_metrics, nout=2)(
-                post_ba_i2Ri1, post_ba_i2Ui1, i2Ti1_expected_graph
-            )
-            pre_ba_inlier_mask_wrt_gt, pre_ba_reproj_error_wrt_gt = dask.delayed(
-                metric_utils.compute_correspondence_metrics, nout=2
-            )(
-                keypoints_i1_graph,
-                keypoints_i2_graph,
-                pre_ba_v_corr_idxs,
-                camera_intrinsics_i1_graph,
-                camera_intrinsics_i2_graph,
-                self._corr_metric_dist_threshold,
-                gt_wTi1_graph,
-                gt_wTi2_graph,
-                gt_scene_mesh_graph,
-            )
-            post_ba_inlier_mask_wrt_gt, post_ba_reproj_error_wrt_gt = dask.delayed(
-                metric_utils.compute_correspondence_metrics, nout=2
-            )(
-                keypoints_i1_graph,
-                keypoints_i2_graph,
-                post_ba_v_corr_idxs,
-                camera_intrinsics_i1_graph,
-                camera_intrinsics_i2_graph,
-                self._corr_metric_dist_threshold,
-                gt_wTi1_graph,
-                gt_wTi2_graph,
-                gt_scene_mesh_graph,
-            )
-        else:
-            pre_ba_R_error_deg, pre_ba_U_error_deg = None, None
-            post_ba_R_error_deg, post_ba_U_error_deg = None, None
-            pre_ba_inlier_mask_wrt_gt, pre_ba_reproj_error_wrt_gt = None, None
-            post_ba_inlier_mask_wrt_gt, post_ba_reproj_error_wrt_gt = None, None
+
+        pre_ba_R_error_deg, pre_ba_U_error_deg = dask.delayed(compute_relative_pose_metrics, nout=2)(
```

Comment:
do these metrics provide the expected output when ground truth is None?

Reply:
They give Nones I think.

Reply:
Yeah just wanted to be sure. Those if statements might have been there for a reason.